README.md

```diff
@@ -13,12 +13,13 @@
 Browsers
 </th><td width=100%>
 
-Load `@octokit/plugin-rest-endpoint-methods` and [`@octokit/core`](https://github.com/octokit/core.js) (or core-compatible module) directly from [cdn.pika.dev](https://cdn.pika.dev)
+Load `@octokit/plugin-rest-endpoint-methods` and [`@octokit/core`](https://github.com/octokit/core.js) (or core-compatible module) directly from [cdn.skypack.dev](https://cdn.skypack.dev)
+ 
```

Comment:
Let me fix it for you 👍🏼 

```suggestion
Load `@octokit/plugin-rest-endpoint-methods` and [`@octokit/core`](https://github.com/octokit/core.js) (or core-compatible module) directly from [cdn.skypack.dev](https://cdn.skypack.dev)
```

Reply:
oh you did not give me access to your fork. Can you apply the suggestion above? You should see a big green button below the "Suggested change"

Reply:
nevermind it worked after all :)